src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -61,8 +61,13 @@ public void ExecuteAnalysis_RequestCannotBeCreated_NoAnalysis()
             var testSubject = CreateTestableAnalyzer(requestFactory: requestFactory.Object);
             testSubject.ExecuteAnalysis("path", "charset", new[] { AnalysisLanguage.CFamily }, Mock.Of<IIssueConsumer>(), analysisOptions, CancellationToken.None);
 
-            testSubject.TriggerAnalysisCallCount.Should().Be(0);
+            testSubject.WaitForFactoryCallToComplete();
+
             requestFactory.Verify(x => x.TryGet("path", analysisOptions), Times.Once);
+
+            // TODO - modify check to be more reliable
+            Thread.Sleep(400); // delay in case the background thread has gone on to call the subprocess
```

Comment:
I couldn't think of simple way of making this reliable without more extensive changes.
I'll revisit it when I do the changes to make the request factories asynchronous.